dashboard/report/generate_json_data.py

```diff
@@ -0,0 +1,12 @@
+def generate_json_data(
+    second_stage: dict, third_stage: dict, fifth_stage: dict
+) -> dict:
+    second_stage.pop("plates_count")
```

Comment:
maybe it'd be good to revise this a little bit? instead of throwing out the things that you do not need, keep the ones that you actually want

\+ imo it'd be good to name the stages instead of enumerating them, i.e.
second_stage -> purging_stage
third_stage -> stats_stage
fifth_stage -> summary_stage


```py
return {
    "purging_stage": {
        "prop1": purging_stage["prop1"],
        ...
    },
    "stats_stage": {
        ...
    },
    "summary_stage": {

    }
}

```